### PR TITLE
CASSANDRA-17881

### PR DESCRIPTION
**Pull request by nonstopdtop**
patch by Chris Thornett, Diogenese Topper; reviewed by -- for CASSANDRA-17881

Co-authored by: Chris Thornett <chris@constantia.io>
Co-authored by: Diogenese Topper <diogenese@constantia.io>

### CODE REVIEW

**Review by ErickRamirezAU — CHANGES_REQUESTED** [review]
The images and captions won't render when I stage the page locally. I've spent 2 hours trying to get it to work but I've given up.

Please fix and test locally before submitting for review. 🙏

**Comment by michaelsembwever** [comment]
Screenshot 2022-09-30 at 13 14 07

--

Screenshot 2022-09-30 at 13 15 07

--

Screenshot 2022-09-30 at 13 15 29


--

Screenshot 2022-09-30 at 13 15 48

--
Screenshot 2022-09-30 at 13 16 07

--
Screenshot 2022-09-30 at 13 16 28

--

Screenshot 2022-09-30 at 13 16 41


--
Screenshot 2022-09-30 at 13 16 57


--
Screenshot 2022-09-30 at 13 17 12

**Comment by michaelsembwever** [comment]
@ErickRamirezAU is it good now?

**Comment by michaelsembwever** [comment]
@dtopdontstop the gracehopper image still looks busted

File: site-content/source/modules/ROOT/pages/blog.adoc [hunk 1]
```diff
@@ -8,6 +8,31 @@ NOTES FOR CONTENT CREATORS
...
 
+//start card
+[openblock,card shadow relative test]
+----
+[openblock,card-header]
+------
+[discrete]
+=== Apache Cassandra Changelog #19
+[discrete]
+==== September 9, 2022
```
Comment (michaelsembwever) [inline site-content/source/modules/ROOT/pages/blog.adoc:19]: fixing this on the fly

**Comment by michaelsembwever** [comment]
merged with 2af01b8f12fe3358f0c5193839b76ebe552ac3d6

**Pull request closed without merging**